kubernetes/update-stack.sh

```diff
@@ -114,37 +114,47 @@ update_data_config() {
 
 delete_influxdb_pvc() {
     echo "WARNING: deleting influxDB data volume"
-
+    set +e
     kubectl delete -f wes-node-influxdb.yaml
     echo "sleep for 3 seconds"
     sleep 3
     echo "deleting Kubernetes pvc: data-wes-node-influxdb-0 and config-wes-node-influxdb-0"
     kubectl delete pvc data-wes-node-influxdb-0 config-wes-node-influxdb-0
+     set -e
+}
+
+cleanup_old_iio_raingauge() {
+    echo "attempting to remove old iio/rainguage plugins"
+    set +e
+    kubectl delete deployment iio-nx iio-rpi raingauge iio-enclosure
+    set -e
 }
 
-# NOTE(Yongho) this is added, but not being used for now
 update_wes_plugins() {
```

Comment:
If we go to the kustomize file approach, then we will need to set many things that are already set by the scheduler (and pluginctl), like https://github.com/waggle-sensor/waggle-edge-stack/blob/main/kubernetes/wes-dev-notebook.yaml#L30. Additionally, it may miss some of upcoming updates from the scheduler and pluginctl if we don’t update the kustomize file accordingly. In other words, launching plugin code can be unified and shared across services.